### ISSUE TITLE
Left Align issues with Web Interface

### ISSUE THREAD

**Issue by sxales** [jellyfin/jellyfin#2583]
The new interface design creates an empty gap down the left side of the screen--under the poster. This wastes space by shoving useful information to the right and therefore truncating it. It also leaves most of the content looking off center. Ideally, all the content starting with Next Up should be left aligned--as it used to be.

System:
 - OS: Windows 10 1909 18363.719
 - Clients: Browser
 - Browser: Firefox 74
 - Jellyfin Version: 10.5.0

Current behavior: 
![interface with gap](https://i.imgur.com/gh5CA1w.jpg)

Expected behavior (mock-up):
![interface without gap](https://i.imgur.com/nb1s2si.jpg)

**Comment by hamburglar2160** [comment]
i believe that this is the intended result. using that dead space would be a good idea for non-mobile interfaces.

my recommendation to fill it would be to get generate a list of other movies in the series or closely related videos. So for example Harry Potter 3 would recommend the other 6 movies. That can be done using already acquired ids through here https://developers.themoviedb.org/3/movies/get-movie-

alternatively... "more from this collection"

**Comment by dkanada** [comment]
@MrTimscampi this is a pretty good idea, although it actually wasn't the intended result.

**Comment by MrTimscampi** [comment]
> @MrTimscampi this is a pretty good idea, although it actually wasn't the intended result.

I agree with using more space, but we need to rework the page a bit to make sure elements don't fight for control of the layout (Making the description and poster part of the header, then the rest of the page a separate element).

We also need the ability to hide part of the description if it's too long, before this makes sense.
